Source/Private/SetAzureServiceConnection.ps1

```diff
@@ -0,0 +1,126 @@
+function SetAzureServiceConnection {
+    param(
+        [string]$Organization,
```

Comment:
Same goes here - Please include a default ParameterSet and parameterSets for parameters.